i18n/en.pot

```diff
@@ -219,6 +219,12 @@ msgstr "Catchment details"
 msgid "Create new catchment areas"
```

Comment:
On line 216/217, I think we should change it to "Catchment area details" for consistency

Reply:
@luongt13 It looks like line 216 still has `Catchment details` instead of `Catchment area details` -- is that intentional?